src/dmx/config.h

```diff
@@ -68,6 +68,13 @@ extern "C" {
 #define RDM_RESPONDER_PIDS_MAX (8 + 16)
 #endif
 
+/** @brief The maximum number of additional parameters that is supported.
+ *  
+ * According to ANSI-E1-20-2010 we can support up to 115 additional parameters.
```

Comment:
The way I've understood ANSI/ESTA E1.20-2010 is that there is no limit to the number of RDM parameters which may be supported. However, only 115 parameters may be sent in a single `RDM_PID_SUPPORTED_PARAMETERS` response. If more than 115 parameters are supported, the `RDM_PID_SUPPORTED_PARAMETERS` response must use `RDM_RESPONSE_TYPE_ACK_OVERFLOW`.

Reply:
Oops you are right. I missed that part of the spec :)